### PR TITLE
[IE CLDNN] Fuse reorder to permute.

### PR DESCRIPTION
**Pull request by yeonbok**
### Details:
Enabled fusing [permute + reorder] to permute

### CODE REVIEW

**Review by vladimir-paramuzov — CHANGES_REQUESTED** [review]

File: inference-engine/thirdparty/clDNN/src/graph_optimizer/remove_redundant_reorders.cpp [hunk 1]
```diff
@@ -278,45 +279,47 @@ void remove_redundant_reorders::run(program_impl& p) {
...
 
-        input.set_output_layout(output_layout, false);
-        if (input.type()->does_possible_implementation_exist(p.get_engine(), input)) {
-            p.replace_all_usages(node, input);
-            p.add_optimized_primitive_info(node.id());
-            p.remove_all_connections(node);
-            p.remove_if_dangling(node);
+            input.set_output_layout(output_layout, false);
+            if (input.type()->does_possible_implementation_exist(p.get_engine(), input)) {
+                printf("found possible implementation\n");
```
[the remark this answers is not in the document]
Comment (yeonbok) [reply inference-engine/thirdparty/clDNN/src/graph_optimizer/remove_redundant_reorders.cpp:315]: opps thank you!

[the remark this answers is not in the document]
Comment (yeonbok) [reply inference-engine/thirdparty/clDNN/src/graph_optimizer/remove_redundant_reorders.cpp]: Fixed

**Review by vladimir-paramuzov — APPROVED** [review]
LGTM. Will be merged after release branch creation

**Comment by vladimir-paramuzov** [comment]
@yeonbok could you please rebase and restart CI?

**Comment by yeonbok** [comment]
> @yeonbok could you please rebase and restart CI?

I did.

**Pull request merged**